src/Uno.Toolkit.UI/Controls/NavigationBar/NativeNavigationBarPresenter.Android.cs

```diff
@@ -52,6 +52,7 @@ private void OnLoaded(object sender, RoutedEventArgs e)
 			if (navBar is { })
 			{
 				Content = navBar.GetOrAddDefaultRenderer().Native;
+				ContentTemplate = null; // normally, the ContentTemplate is inherited from the NavigationBar, but in this case, we don't want it to. We want to use the renderer directly as the child.
```

Comment:
@ramezgerges / @kazo0 do you think it is possible to add more tests for this?

Reply:
We currently have tests that should cover it I believe. We caught this because those tests were failing in the canary

Reply:
if we dont want _automagic_ content-template inheritance, 
just subclass from ContentControl instead of ContentPresenter,
since the latter have special interactions with its containing ContentControl
which isnt desired here